doc/storage/appendix/sra.rst

```diff
@@ -0,0 +1,919 @@
+
+   .. unmitigated:: DM.AUTHORISED
+      :impact: VH
+      :likelihood: L
+
+   .. residual:: DM.AUTHORISED
+     :impact: VH
+     :likelihood: VL
+
+   .. unmitigated:: DM.SECURE_CHANNEL
+      :impact: VH
+      :likelihood: L
+
+   .. residual:: DM.SECURE_CHANNEL
+     :impact: VH
+     :likelihood: VL
+     
+   .. mitigations:: :mitigation:`GlitchDetection` in all deployment models, active glitch detection circuits can raise an exception if a glitch is detected, permitting the computing circuitry to take corrective action. 
+   
+
+.. threat:: Glitching during a write
+   :id: GLITCH_WRITE
+
+   .. description:: An attacker with physical access might be able to disrupt the power to prevent a write from being completed.
+
+   .. adversarial-model:: `AM.3`
+
+   .. security-goal:: `SG.INTEGRITY`
+   
+   In this threat, an attacker with physical access to the device causes a power or frequency glitch to cause a write to fail.
+   
+   In `DM.PROTECTED` the API does not provide a mechanism to detected this. Therefore, the implementation must provide alternative means to detect glitches. 
+   
+   In `DM.EXPOSED` the invalid write will be detected on a future read provided the system uses `M.MAC`, which it should do due to other threats. 
+   
+   In `DM.AUTHORISED` and `DM.SECURE_CHANNEL` there is a brief time of check, time of use (TOCTOU), window, where the Storage Medium has verified the command but has not written the data to physical storage. In this case, when a subsequent read occurs, the Storage Medium will apply a new tag to the reply, and the Storage Service will not be aware that it is returned a corrupted read. This risk should be TRANSFERRED to the Storage Medium which should offer glitch detection.
+   
+  
+   .. unmitigated:: DM.PROTECTED
+      :impact: VH
+      :likelihood: H
+            
+   .. residual:: DM.PROTECTED
+      :impact: VH
+      :likelihood: L
+      
+   .. unmitigated:: DM.EXPOSED
+      :impact: VH
+      :likelihood: H
+
+   .. residual:: DM.EXPOSED
+      :impact: VH
+      :likelihood: VL
+
+   .. unmitigated:: DM.AUTHORISED
+      :impact: VH
+      :likelihood: H
+
+   .. residual:: DM.AUTHORISED
+     :impact: H
+     :likelihood: VL
+
+   .. unmitigated:: DM.SECURE_CHANNEL
+      :impact: VH
+      :likelihood: H
+
+   .. residual:: DM.SECURE_CHANNEL
+     :impact: H
+     :likelihood: VL
+
+   .. mitigations:: in all deployment models `M.GlitchDetection` can be used to reduce the risk of a successful glitch. 
+   
+      :mitigation:`ReadAfterWrite`, in all Deployment models, the Storage Service can perform a read operation immediately after a write, while it still retains the original value in memory and compare the two before confirming the write to the caller. However, this has performance challenges, Therefore the implementation may  decide to do this on a smapling basis. 
+    
+.. _sra-mitigations:
+
+Mitigation Summary
+------------------
+
+.. list-table:: Mitigations 

```

Comment:
I like this by-DM summary. For this API, with very distinct implementation designs, this provides a valuable view of the mitigations.